src/coreclr/src/utilcode/ex.cpp

```diff
@@ -1269,7 +1269,7 @@ static DWORD MarkAsThrownByUsWorker(UINT numArgs, /*out*/ ULONG_PTR exceptionArg
     exceptionArgs[0] = arg0;
 
 #if !defined(FEATURE_UTILCODE_NO_DEPENDENCIES)
```

Comment:
`#if !defined(FEATURE_UTILCODE_NO_DEPENDENCIES) && defined(HOST_WINDOWS)` ?

Reply:
I tried ignoring this assignment on Unix and just returning TRUE or FALSE from `WasThrownByUs`. Either way leads to failures. It looks like this codepath is active on Unix. I did not look deeply what we do here, but we have to assign. It does not matter what, as long as it is not 0 and matches what `WasThrownByUs` expects.

I'll use `#define  THROWN_BY_US_MARKER   ULONG_PTR(-1)`